ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/workloadCveOverviewPage.test.js

```diff
@@ -41,67 +40,46 @@ describe('Workload CVE overview page tests', () => {
         if (!hasFeatureFlag('ROX_WORKLOAD_CVES_FIXABILITY_FILTERS')) {
             this.skip();
         }
+
         visitWorkloadCveOverview();
 
         // We want to manually test filter application, so clear the default filters
         cy.get(selectors.clearFiltersButton).click();
-        cy.get(selectors.hiddenSeverityCount('Critical')).should('not.exist');
-
-        // Get the first CVE row from the table with a non-zero severity count for -any- severity
-        cy.get(selectors.nonZeroImageSeverityCounts)
-            .first()
-            .then(($severityCount) => {
-                const [nonZeroSeverity, unusedSeverities] = extractNonZeroSeverityFromCount(
-                    $severityCount.attr('aria-label')
-                );
+        cy.get(selectors.isUpdatingTable).should('not.exist');
+
+        const urlBase = '/api/graphql?opname=';
```

Comment:
For your info, not a change request: see `graphql` helper function:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/constants/apiEndpoints.js#L1

Reply:
Oh nice, I'll refactor this the next time I visit this test to use the helper.